packages/openapi-parser/README.md

```diff
@@ -55,6 +55,30 @@ const specification = `{
 const { schema, errors } = await dereference(specification)
 ```
 
+### Bundle external references
+
+The OpenAPI specification allows to point to external files (URLs or files). But sometimes, you just want to combine all files into one.
+
+```ts
+import { bundle } from '@scalar/openapi-parser'
+
+const document = {
+  openapi: '3.1.0',
+  info: { title: 'Bundled API', version: '1.0.0' },
+  paths: {},
+  components: {
+    schemas: {
+      User: { $ref: './user-schema.json#' }
+    }
+  }
+}
+
+// This will resolve and inline all external $refs (like user-schema.json)
+await bundle(document)
+
+console.log(document)
+```
+
```

Comment:
can you add examples here?

* using the fetchUrls plugin (could be in a browser)
* using the fetchUrls & readFiles plugin (only in Node.js)
* changing the limit of concurrent requests